c2corg_ui/static/js/style/stickyfilters.js

```diff
@@ -0,0 +1,55 @@
+goog.provide('app.stickyFilters');
+
+goog.require('app');
+
+
+/**
+ * 
+ * @export
+ */
+app.stickyFilters = function() {
+  $('.documents-list-section').scroll(function() {
+    adaptFiltersWidth();
+    var scrollMax = $('.simple-filters').height() + 30;
+   
+   // if you scroll down and there is no sticky bar, add it
+    if ($(this).scrollTop() > scrollMax && $('.sticky-filters-btn-container').length === 0) {
+      $('.more-filters-btn-container').addClass('sticky-filters-btn-container');
+      $('#moreFilters').addClass('sticky-filters-moreFilters');
+      $('.simple-filters').clone().prependTo('#moreFilters');
+      //show only part of filters on small screen
+      if (window.innerWidth < 620) {
+        $('#moreFilters').addClass('overflow-scroll');
+      }
+    }
+    //when you scroll up, remove the cloned filters and the sticky classes
+    if ($(this).scrollTop() < scrollMax && $('.sticky-filters-btn-container').length > 0 && $('.simple-filters').length <= 2) {
+      var simpleFilters = $('#moreFilters .simple-filters').clone();
+      $('.more-filters-btn-container').removeClass('sticky-filters-btn-container');
+      $('#moreFilters').removeClass('overflow-scroll sticky-filters-moreFilters');
+      $('.simple-filters').remove();
+      $('.filters').prepend(simpleFilters);
+    }
+  });
+
+  $(window).resize(function() {
+    adaptFiltersWidth();
+    if (window.innerWidth > 620 && $('.filters').css('display') === 'none') {
+      $('.filters').show();
+    }
+  });
+   // show/hide filters on click, only available on phone
+  $('.show-documents-filters-phone').click(function() {
+    $('.filters').toggle();
+  });
+
+  function adaptFiltersWidth() {
+    var width = $('.documents-list-section').width();
+    // sticky bar width = filters width
+    if ($('#moreFilters').hasClass('sticky-filters-moreFilters')) {
+      $('#moreFilters').css('width', parseInt(width, 10));
+    } else {
+      $('#moreFilters').css('width', parseInt($('.filters').width(), 10));
```

Comment:
you mean  $('#moreFilters').css('width', parseInt(width, 10)); ?
Unfortunately no, the position of this element is absolute, it floats above the divs and 100% never fills up the space left but is more or less, in addition it is always different than it should when you resize. At least I have no idea how to change it with css only :P